homework/index.js

```diff
@@ -0,0 +1,64 @@
+window.onload = () => {
```

Comment:
If you're not using this, remove again. But it's good practice to put the 'initialization code' here - or better yet, move that code in one of more functions, and call those!